### PR TITLE
docs: update comparison

### PR DESCRIPTION
**Pull request by aeneasr**
First of all, congratulations on publishing a  successful open source project :) It looks like you are solving a real pain in your realm and it's awesome to see it being published in the open!

With this PR I want to clarify a few things in the comparison which I believe are unfair to Ory. I will try to add explanations for every point. I think the comparison to Ory Hydra directly is unfair, because Ory Hydra is an Authorization Server, not an identity provider. This project here however is an identity provider and (as far as I understand) reverse proxy. Ory has these capabilities as well, and they are the projects Ory Kratos, Ory Oathkeeper, and in parts Ory Hydra. That's why I changed the description to "Ory".

- Simple: Self-hosting Ory is surely not an easy task due to the service oriented architecture. However, using Ory Network you can get started in seconds with a fully functioning auth system
- Anonymous clients: Partially correct - Ory supports OIDC Dynamic Client Registration which is essentially a standard to allow anyone to create and manage clients. Functionally, it works different from the proposal here. From a use case view, it's pretty much the same thing
- Multi-domain auth: Will definitely take a look how you solved this
- Passwordless email login, MFA: Supported natively by Ory Kratos
- HTTP API: Ory has both gRPC and REST
- Trusted header auth: Supported by Ory Oathkeeper
- Upstream OIDC/OAUth2: Core functionality of Ory
- LDAP/SAML: Due to the modular architecture of Ory this can relatively effortlessly be added by writing a bit of code
- Admin GUI: Core functionality of Ory Network

I hope this helps understanding the reason for the PR. I wish you all the best with your project and if you have any questions related to auth feel free to drop me a slack message in our slack :)

### CODE REVIEW

**Comment by jedahan** [comment]
If you have to setup 3 different pieces of software (kratos, oathkeeper, and in parts hydra) to achieve the same goal I don't know if I'd qualify that as simple.

**Comment by aeneasr** [comment]
> If you have to setup 3 different pieces of software (kratos, oathkeeper, and in parts hydra) to achieve the same goal I don't know if I'd qualify that as simple. Oh wait 4 with Ory Network.

Quoting myself:

> simple: **Self-hosting Ory is surely not an easy task due to the service oriented architecture**. However, using Ory Network you can get started in seconds with a fully functioning auth system

Ory Network is the SaaS product combining the individual services into one coherent offering and it's literally two clicks to set it up with a nice UI, or using two commands with the Ory CLI. I'd say that qualifies as simple.

If the comparison is about self-hosting (it's not, it just says "simple"), then definitely setting up one service is significantly easier than 3 or more! If it's about general product offering, I'd disagree with your assumption.

**Comment by jedahan** [comment]
The description of the project, and the introduction both include in the first sentence “ designed for self-hosters” so yeah, I would assume the expectation of people interested in the comparison matrix have that context in mind.

**Comment by jedahan** [comment]
Thank you for correcting my mistake of what Ory Network was. So self hosting only requires setting up 3 services, not 4.

File: README.md [hunk 1]
```diff
@@ -193,27 +193,24 @@ incomplete and probably incorrect in many cases. If you have a correction,
...
+|--------------------------|--------------------------------------------------------|---------------------------------------|--------------------------------------|---------------------------------------|-----------------------------------------------|--------------------------------------------------------------|---------------------------|----------------------------|---------------------------------|---------------------------------|
+| Simple                   | ✅                                                      | ✅                                     | ❌                                    | ❌                                     | ❓                                             | ❓                                                            | ❓                         | ✅                          | ❓                               | ❓                               |
```
Comment (aeneasr) [inline README.md:198]:
```suggestion
| Simple to self-host                  | ✅                                                      | ✅                                     | ❌                                    | ❌                                     | ❓                                             | ❓                                                            | ❓                         | ❌                          | ❓                               | ❓                               |
```

**Comment by aeneasr** [comment]
Added a suggestion according to your feedback - thanks!

**Comment by anderspitman** [comment]
Also, are you aware of any popular client apps that implement dynamic client registration? One big advantage of anonymous auth is that it works with existing clients.

**Comment by anderspitman** [comment]
> An initial token is not required, and the feature is optional. Honestly, there are no real differences or advantages between either approach other than that OIDC Dynamic Client Registration is a public spec. However, the approach obligator takes I think is totally fine for the sake of simplicity / ease of use.

Excellent, I'll merge this and add a new row for dynamic client registration. Thanks again!

**Comment by anderspitman** [comment]
@aeneasr [this](https://www.ory.sh/docs/hydra/guides/oauth2-clients#openid-dynamic-client-registration) seems to indicate that an access token is required for dynamic client registration. Can you link to documentation that shows how to do it without a token?

**Comment by aeneasr** [comment]
It's not required for registration, the token is however needed to perform updates to the client. You can also do it completely without tokens using the Admin API: https://www.ory.sh/docs/reference/api#tag/oAuth2/operation/createOAuth2Client

**Pull request merged**